### DISCUSSION TITLE
Links to CLI are broken in User Guide

### DISCUSSION THREAD

**Discussion by mjalkio** [pawelsalawa/sqlitestudio#4379] [Q&A]
### Details
I am supporting some students that are attempting to use this tool. They are reporting issues with importing a CSV file that contains blank lines (I'll ask them to submit a bug report for this, since I don't have all the details). I recommended that they try the `.import` command instead, but it returns errors. Issue https://github.com/pawelsalawa/sqlitestudio/issues/2787 suggests that dot commands are only supported by the CLI and not the GUI. The [user guide](https://github.com/pawelsalawa/sqlitestudio/wiki/User_Manual#command-line-interface-cli) mentions the CLI and links to https://github.com/pawelsalawa/sqlitestudio/wiki/CLI "for more details." However, that link just redirects back to https://github.com/pawelsalawa/sqlitestudio/wiki/

Does the CLI exist? Where is it documented?

### Steps to reproduce
Click the "See [Command line interface](https://github.com/pawelsalawa/sqlitestudio/wiki/CLI) for more details." link in the [user guide](https://github.com/pawelsalawa/sqlitestudio/wiki/User_Manual#command-line-interface-cli).

### Operating system
N/A

### SQLiteStudio version
N/A

**Comment by pawelsalawa** [comment]
@dingyiduan7 this is not a place to ask an arbitrary question. I've moved your question to #4378. Let's continue there.

**Comment by pawelsalawa** [comment]
Sorry @dingyiduan7 - I misinterpreted your posts. It appears that they are connected to this original post above from @mjalkio 
So...

You guys are mistakening SQLiteStudio CLI with the official SQLite3 CLI (https://sqlite.org/download.html). The `.import` command that you are trying to execute does not exist in SQLiteStudio - neither GUI or CLI.

**Reply by mjalkio** [reply]
I'm confused about why you closed the issue. Are you saying that the link to https://github.com/pawelsalawa/sqlitestudio/wiki/CLI in your documentation is not broken?

**Reply by pawelsalawa** [reply]
It's not broken. The page was just never created and is pending creation. It's like an article on wikipedia that awaits creation.

**Reply by mjalkio** [reply]
Maybe you could create a placeholder page? If you visit a Wikipedia page that awaits creation it tells you that the page does not exist: https://en.wikipedia.org/wiki/ThisPageDoesNotExist

**Reply by pawelsalawa** [reply]
Okay, I wrote few words there to explain.

**Comment by dingyiduan7** [comment]
Hi,

Yes my post was continued on for @mjalkio.

But does the importing through GUI seem to have a problem though when there
was a blank row(s)?

On Thu, Jan 20, 2022 at 01:14 Paweł Salawa ***@***.***> wrote:

> Sorry @dingyiduan7 <https://github.com/dingyiduan7> - I misinterpreted
> your posts. It appears that they are connected to this original post above
> from @mjalkio <https://github.com/mjalkio>
> So...
>
> You guys are mistakening SQLiteStudio CLI with the official SQLite3 CLI (
> https://sqlite.org/download.html). The .import command that you are
> trying to execute does not exist in SQLiteStudio - neither GUI or CLI.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pawelsalawa/sqlitestudio/discussions/4379#discussioncomment-2004533>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASDTGPEAPNS7SGBVCCA2QW3UW7G57ANCNFSM5MMFUFLQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>
-- 
Best Regrads,

Dingyi Duan

(716)-866-9663

***@***.***

**Reply by pawelsalawa** [reply]
It seems that your CVS file has some lines, that have just single value (and your `sales` table has total of 10 columns). If the value for a column is not provided in a CSV line, SQLiteStudio assumess `NULL`. This unfortunately violates the constraint that you have on the `listid` column (being `NOT NULL`).
